src/Classical/linear_code.jl

```diff
@@ -407,6 +407,7 @@ function information_set(C::AbstractLinearCode)
     return pivot_inds
 end
 
+#TODO doc
```

Comment:
"""
random_information_set(C::AbstractLinearCode)

Return a set of column indices which contain an information set for `C`.
"""

need to put a tab before the function name for the documentation library^